lib/fluent/timezone.rb

```diff
@@ -100,15 +111,15 @@ def self.formatter(timezone = nil, format = nil)
         case
         when format.is_a?(String)
           return Proc.new {|time|
-            Time.at(time).localtime(offset).strftime(format)
```

Comment:
If refinements is going to be removed, I think these lines(L103, L107, L111, L127, L131, and L135) should be left as it is.
so when removing refinements in the future, we need not change these lines (and it'll be able to narrow down the scope of refinements)

Reply:
fluentd v2 will drop Integer support for event time, so we can call `EventTime#to_time` without Integer ext.

Reply:
I see.